### PR TITLE
Scientist Jobs Split

### PR DESCRIPTION
**Pull request by Apogee-dev**
[role]
Separates the xenoarchaeology and xenobiology alt titles off from the scientist job, making them new, independent jobs that do not share slots with the Scientist job, similar to Virologist being split off from MD. Xenoarchaeologists spawn in the break room and research shuttle hallway, xenobiologists spawn in xenobiology or in the break room depending on the map.

:cl:
 * rscadd: Added Xenoarchaeologist (with Anomalist alt title) and Xenobiologist as independent jobs.
 * rscdel: Removed xenoarch and xenobio-related alt titles from the Scientist job

### CODE REVIEW

**Comment by gurfan** [comment]
why don't you just change the map spawns

**Comment by Apogee-dev** [comment]
> why don't you just change the map spawns

I don't really know anything about mapping yet and didn't want to wrap a bunch of map changes into this PR to minimize potential conflicts.

**Comment by gurfan** [comment]
This is stupid without the mapping changes. There are no other mapping PRs open right now and the changes needed are small

**Comment by Apogee-dev** [comment]
> This is stupid without the mapping changes. There are no other mapping PRs open right now and the changes needed are small

Fair enough. Guess I'll finally have to bite the bullet and actually learn some basic mapping. I'll add map changes when i'm done, for Box at a bare minimum

**Comment by Eneocho** [comment]
Remember to modify access accordingly and possibly give some add some Priority rewards to the new jobs like on #29367.

**Comment by Apogee-dev** [comment]
> Remember to modify access accordingly and possibly give some add some Priority rewards to the new jobs like on #29367.

I'm not planning on modifying access, and, reasonably speaking, I couldn't anyway- xenoarchaeologist still needs RnD access to actually research anomaly items, and scientist still needs xenoarchaeology access to get to toxins and xenobotany on most maps. The only real contender for restricted access here is xenobiologist, but I don't really want to mess with that.

Good tip on the priority rewards, though, I'd forgotten about those despite being part of the initial plan.

**Comment by Eneocho** [comment]
> 
> 
> > Remember to modify access accordingly and possibly give some add some Priority rewards to the new jobs like on #29367.
> 
> I'm not planning on modifying access, and, reasonably speaking, I couldn't anyway- xenoarchaeologist still needs RnD access to actually research anomaly items, and scientist still needs xenoarchaeology access to get to toxins and xenobotany on most maps. The only real contender for restricted access here is xenobiologist, but I don't really want to mess with that.
> 
> Good tip on the priority rewards, though, I'd forgotten about those despite being part of the initial plan.

For the xenobiologist, you could give a bottle with 1u of slimetox, so the xenobio doesn't have to rush green to avoid getting glomped. (Though to be fair, who would in their right mind priority request a xenobio? Those new slime based upgrades would make it tempting for engi and mining though).

**Comment by Apogee-dev** [comment]
Added spawnpoints on all maps. I'm skipping out on priority care packages for this PR, because it turns out scientist and RD don't get unique care packages either, so xenoarch and xenobio players aren't really losing anything by not having one. Not to say the science jobs couldn't use unique priority rewards along the lines of the ones added to medbay jobs some time ago, but that's something for another PR rather than a requirement for job parity like I'd thought.

**Comment by Eneocho** [comment]
How didja map the things and PR em appropiately? Just edit the map on strongdmm and upload?

**Comment by Apogee-dev** [comment]
> How didja map the things and PR em appropiately? Just edit the map on strongdmm and upload?

yeah, pretty much that. honestly, i was expecting something to go wrong.

**Comment by Eneocho** [comment]
> 
> 
> > How didja map the things and PR em appropiately? Just edit the map on strongdmm and upload?
> 
>i was expecting something to go wrong.

Same. It's the reason I didn't map my carp breeding in xenobio PR and left it as adminbus for the time being. I'll map it after yours gets merged.

**Comment by DeityLink** [comment]
Finally!

**Comment by chaklett** [comment]
YES!!!!!!!!!!!!!!!!!! SINGLEPLAYER JOB IS NOW OFFICIALLY SINGLEPLAYER!!!!

**Comment by BomberBro** [comment]
Twice the roles not doing research, this pretty much leaves the "scientists" to xenobotany if they want to science

**Comment by Apogee-dev** [comment]
> Twice the roles not doing research, this pretty much leaves the "scientists" to xenobotany if they want to science

>implying xenobiologists or xenoarchaeologists actually do RnD in the current state of the game beyond whatever they personally need

there's a reason i'm not changing access for any of these jobs and don't plan to anyway. scientists can still go do xenobio or xenoarch if they get bored, and xenobio/xenoarch can still do RnD if hell freezes over and they feel like contributing to the station. or if the crew decides to lynch them, like with mechanics that refuse to set up power. the difference is that the xenobio/xenoarch players won't eat up scientist job slots, you can't get a sci department that's just five xenoarchs and nothing else without opening up extra xenoarch slots, and a HoP that prioritizes scientists won't get a gaggle of xenoarchs that don't actually do science instead

**Comment by BomberBro** [comment]
I just don't really see the point then, this isn't like an atmos tech/engineer split where each guy gets a completely different locker, hardsuit, and responsibilities, it'd be like splitting up engineer into SME man, repair man, and Singulo man as job titles. Also this technically increases the science job count by 4 which I don't really think it's beneficial to the space ecosystem

**Comment by Apogee-dev** [comment]
> I just don't really see the point then, this isn't like an atmos tech/engineer split where each guy gets a completely different locker, hardsuit, and responsibilities, it'd be like splitting up engineer into SME man, repair man, and Singulo man as job titles. Also this technically increases the science job count by 4 which I don't really think it's beneficial to the space ecosystem

It's more like splitting virologist off from medical doctor. They both have the same access and can both theoretically do each others' job, but somebody taking viro means they plan on spending the vast majority of the round holed up in their jobbie cave. The same is true of both xenobio and xenoarchaeologist- with xenoarch spending all round on the roid instead. By splitting up those jobs from scientist, you can put a limit on how many players are doing that, or if nothing else are _expected_ to be doing that. 

Xenoarchs also typically have a completely different toolset from a standard scientist, with their unique suits, belts, and other tools, and there's a limit on how many of those tools are available anyway- for most maps, only two or three xenoarch players can actually get a full toolkit with excavation suit, any extras wind up twiddling their thumbs or doing other science tasks.

**Comment by BomberBro** [comment]
>but somebody taking viro means they plan on spending the vast majority of the round holed up in their jobbie cave.

I'm of the opinion that singleplayer jobs are a mistake except on lowpop, this is a multiplayer game built on roleplay at its heart and encouraging singleplayer makes shit feel empty even on 20-30 pop. 

>That, and there's a limit on how much xenoarch gear each map has.

Pretty easy to print the exca drill, after about 30 minutes you could outfit an army of xenoarchs. I'm worried that this will create a "I'm a mechanic, it's not my job to set up power" tier waste of a job slot where the people playing never actually have to interact with the station or fellow players.

**Comment by Apogee-dev** [comment]
> I'm of the opinion that singleplayer jobs are a mistake except on lowpop, this is a multiplayer game built on roleplay at its heart and encouraging singleplayer makes shit feel empty even on 20-30 pop.

I actually agree with you on this in that too many players on singleplayer jobs can heavily affect station dynamics, and I'd be in favor of changes that rule certain jobs out based on population, but that's a much larger balance discussion than just xenoarch and xenobio.

> Pretty easy to print the exca drill, after about 30 minutes you could outfit an army of xenoarchs. 

It's mostly the excavation suits that are the bottleneck here. Sure, you can get normal space suits, but getting extra hardsuits in general is usually pretty tricky without plundering another department. You can't print them and you can't order more (actually, there's an idea- maybe roboticists should be able to build hardsuits and handle hardsuit upgrades?)

>I'm worried that this will create a "I'm a mechanic, it's not my job to set up power" tier waste of a job slot where the people playing never actually have to interact with the station or fellow players.

To be fair, the playerbase already comes down pretty hard on mechanics that pull that shit. That's why we had the bomb collar thing a while back after all. I wouldn't be opposed to collaring xenoarchs who refuse to do RnD either, and it's not like they don't already act like that. The xenoarch players that are stubborn shits won't get any more or less stubborn, and the helpful ones aren't likely to change either.

**Comment by BomberBro** [comment]
>maybe roboticists should be able to build hardsuits and handle hardsuit upgrade

Dear god no, I don't know if the powers that be would allow it now but historically science not being able to make space suits was held up as one of the few things they were lacking (in a good gameplay balance sort of way). I think they should be orderable in cargo after like an hour or two in the round though.

>And it's not like they don't already act like that.

See, I think mechanics do this because there's already a job that's "supposed" to set up the engine, so by picking mechanic instead of that they have an excuse to not set anything up because they never "intended" to anyway. If somebody goes scientist, doing RnD is like their only real responsibility, they go into the role knowing that, and thus they don't have a real excuse if they decide to leave and never deal with it.

**Comment by Apogee-dev** [comment]
> I think they should be orderable in cargo after like an hour or two in the round though.

Good idea, better than being able to build them. I still like the idea of robos handling hardsuit upgrades and having a hardsuit upgrade station in the lab or mechbay though. Maybe allow people to order hardsuit frames that need finishing at robotics to get a specific hardsuit variant or something. Since you have to order the upgrade station right now it's usually cargo handling upgrades or just the miners taking advantage of them, which is kind of weird.

> If somebody goes scientist, doing RnD is like their only real responsibility, they go into the role knowing that, and thus they don't have a real excuse if they decide to leave and never deal with it.

Alt titles already function like something of a "statement of intent" if you ask me. That's probably _why_ xenoarch players may refuse to do RnD for other departments or, god forbid, answer requests at the RnD desk, as things already are- by taking the xenoarch title they're already saying "I'm gonna go break rocks and come back with a crate full of gunk." That, and the fact that going out there and breaking rocks means it takes a xenoarch a good five minutes to haul ass back to RnD and handle peoples' requests even if he actually wants to be helpful. Miners have a similar habit of limiting the number of trips they have to take back to the station until they've dug up enough resources to keep other departments satisfied and actually have the free time to hang around on the station or go explore space, because running back and forth from the station to the roid is a hassle. That doesn't usually stop miner players from taking cargo orders if the department isn't staffed, though.

**Comment by Eneocho** [comment]
There's an argument to be made around the idea of stronger jobs being by design more complicated. You can do all sorts of wacky shit with telecomms, which is why it's arguably the most complex area to work with. Same with botany. You can do the simple botanyjob and get the functional but otherwise unremarkable plants or do the comparatively complex shit and get the retarded strong plants. Or engineering. You can do the simple baby's first engine TEG/AME and get some stable power or go full autismo analysis and make an optimized, partially delaminated, autismo plasma shard that shits out 300GW per second. 

The problem with that, is that 2 of the strongest jobs (In overall capacities, both crew and traitorwise) are really fucking simple, to the point of being vapid as fuck. You HAVE to do the meta, or you won't get to the (((fun))) (for you) part. There's this thing about all science jobs being soulless voids that encourage singleplayer powergaming (With possibly the exception of Scientist proper, which while still metagamey, is not singleplayer). 

You don't wanna rush cerulean, green and purple slimes? Enjoy either dying at the hands of the slimes or not getting to your personal objective before the shuttle gets inevitably called. You don't wanna powergame retardmechs? Enjoy doing literally nothing because almost nobody plays borg/AI anymore. You want to get them genetic powers? Enjoy that gaschamber you just built, because it's that or sitting on the jobbie hole for 40 minutes. You want some wacky artifacts? You damn better rush that mercury digsite or metagame with telescience to get the anomaly research.

Oh, you want the godmaterial that can make the uber mech AND make you nigh immortal? Sure, just grind this RNG gacha for 10 minutes. What's that? You want to rush the crew and kill them fast? No worries, gold slimes are only 2 slime divisions away, you'll get the gunkbutton in no time!

The only science job that doesn't completely fall into that problem is Scientist, which is still very soulless due to the fact that it is literally following a checklist for 10 minutes then doing nothing for the rest of the round.

Don't get me wrong, I do find xenoarch (and some parts of science) to be interesting as a concept, but it is soulless in all the worst ways, not only it encourages singleplayer metagaming, but also mass gunk with the retard artifacts, retard common larges and those FUCKING CLAWS GODDAMN SOMEONE REMOVE THE CLAWS' PORTAL RIFTS JESUS FUCKING CHRIST. It's dumb easy for anyone to go xenoarch mine for 20 minutes and return with round-ending gunk artifacts/guns (don't tell me about muh triangulation, we all know you only have to meme for 10 minutes digging small artifacts to get enough for the advanced digsite locator/alden saraspova). It's a gamestyle that circles around metagaming and powergaming by design, you can get those larges and have the (((fun))) BEFORE the antag forces a shuttle call if you metagame the digsites. It's a gamestyle heavily contrasted with the miners' gamestyle, which revolves around exploring, powerful but vulnerable. You can't metagame your way out of getting cucked by the roidmobs (we all know about that one trick, don't mention it though).

While miners and xenoarchs are similar in their singleplayer-like job, there's a whole different playerbase (and playerstyle?) for both roles. It's a surprise to see a xenoarch doing proper RnD or answering RnD requests (I've seen it twice ever, and one of the two guys just moved the machinery to a common area so he wouldn't be bothered further). On the other hand, it is quite common to see a miner working as Cargo Tech 2: now with drills. There's a reason we have the whole thing about miners rushing artifact zones to *dab on xenoarcheologists. There's some fake sense of superiority implicit in playing xenoarch, facilitated by how easy it is to play now and getting the shit to literally end the round.

TL;DR: Science is absurdly simple and way too soulless and should be more complex to match its potential.

**Comment by Apogee-dev** [comment]
> TL;DR: Science is absurdly simple and way too soulless and should be more complex to match its potential.

I'm not sure I see how the points you're making here are particularly relevant to this PR. Science's lack of SOVL would be an issue for a PR that, say, adds content to the science jobs or overhauls their core systems, and that isn't really what I'm setting out to do here.

**Comment by Eneocho** [comment]
> 
> 
> > TL;DR: Science is absurdly simple and way too soulless and should be more complex to match its potential.
> 
> I'm not sure I see how the points you're making here are particularly relevant to this PR. Science's lack of SOVL would be an issue for a PR that, say, adds content to the science jobs or overhauls their core systems, and that isn't really what I'm setting out to do here.

Honestly, I was just ranting.

**Comment by DeityLink** [comment]
Many such cases.
